macosx/hbsign

```diff
@@ -146,59 +163,55 @@ for TARGET in "${@}"; do
+        verify "${TARGET}"/Contents/Frameworks/Sparkle.framework
 
-        echo "  Validating Updater.app"
-        codesign --verify -vv "${TARGET}"/Contents/Frameworks/Sparkle.framework/Versions/Current/Resources/Updater.app >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
+        echo "  Validating Updater App"
+        verify "${TARGET}"/Contents/Frameworks/Sparkle.framework/Versions/Current/Resources/Updater.app
 
         echo "  Validating XPC Services"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/HandBrakeXPCService.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/HandBrakeXPCService2.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/HandBrakeXPCService3.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/HandBrakeXPCService4.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/org.sparkle-project.Downloader.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/org.sparkle-project.InstallerConnection.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/org.sparkle-project.InstallerLauncher.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-        codesign --verify -vv "${TARGET}"/Contents/XPCServices/org.sparkle-project.InstallerStatus.xpc >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-    fi
+        for FILE in "${XPC[@]}" "${XPC_EXT[@]}"; do
+            verify "${FILE}"
+        done
 
-    echo "  Validating Bundle"
-    codesign --verify --deep --strict --verbose=4 "${TARGET}" >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
+        echo "  Validating App Bundle"
+        verify "${TARGET}" "deep"
 
-    if [[ "${TARGET##*/}" != 'HandBrakeCLI' ]]; then
         echo "  Validating Execution Privileges"
         spctl -a -t exec -vv "${TARGET}" >>"${LOG}" 2>&1 || exit_with_error 1 "Validation failed. More info may be available in ${LOG}"
-    fi
+    elif [[ "${TARGET##*/}" == 'HandBrakeCLI' ]]; then
+        echo "  Signing Executable"
+        sign "${TARGET}" "cli"
+
+        echo "  Validating Executable"
+        verify "${TARGET}" "deep"
+    else
+        exit_with_error 1 "This script does not know how to sign the specified target.'
```

Comment:
I think there should be a quotation marks at the end instead of '

Reply:
Oof, thanks for spotting that. Pushed an additional commit with the fix.